script.sh

```diff
@@ -147,7 +146,41 @@ case "${TRAVIS_DEBIAN_DISTRIBUTION}" in
 		;;
 esac
 
-# TRAVIS_DEBIAN_DISTRIBUTION is now set, so set dependent options
+# detect derivatives
+if [ "${TRAVIS_DEBIAN_DERIVATIVE:-}" = "" ]
+then
+	# strip derivate common pockets
```

Comment:
Pockets?

Reply:
yeah, ubuntu and mint are using pockets there. I don't know which term (not vague) to use. See the answer below

Reply:
Well, please use a consistent term! .. And given this script is mostly for Debian, please use that one.